packages/spacex_api/lib/src/spacex_api_client.dart

```diff
@@ -46,7 +46,36 @@ class SpaceXApiClient {
   /// REST call: `GET /rockets`
   Future<List<Rocket>> fetchAllRockets() async {
     final uri = Uri.https(authority, '/v4/rockets');
+    final responseBody = await _apiRequest(uri);
 
+    try {
+      return responseBody
+          .map((dynamic item) => Rocket.fromJson(item as Map<String, dynamic>))
+          .toList();
+    } catch (_) {
+      throw JsonDeserializationException();
+    }
+  }
+
+  /// Fetches all SpaceX crew members.
+  ///
+  /// REST call: `GET /crew`
+  Future<List<CrewMember>> fetchAllCrewMembers() async {
+    final uri = Uri.https(authority, '/v4/crew');
+    final responseBody = await _apiRequest(uri);
+
+    try {
+      return responseBody
+          .map((dynamic item) =>
+              CrewMember.fromJson(item as Map<String, dynamic>))
+          .toList();
+    } catch (_) {
+      throw JsonDeserializationException();
+    }
+  }
+
+  // TODO: HOW DO YOU DESCRIBE private METHODS?
+  Future<List<dynamic>> _apiRequest(Uri uri) async {
```

Comment:
```suggestion
  Future<List<dynamic>> _get(Uri uri) async {
```

I'd recommend renaming to `_get` to indicate a GET request is being made.